notebooks/community/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -0,0 +1,1272 @@
+{
```

Comment:
<p>Explain what the dataset predicts.</p><p> </p><p>You will learn -&gt; You learn</p><p> </p><p>endpoint on Vertex AI -&gt; Vertex AI Endpoint</p><p> </p><p>I would consider dropping <code>is aimed at data analysts and data scientists </code>.Seems to unintentional narrow the target audience  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
> Explain what the dataset predicts.
Added: `This dataset will be used to predict which users will, after 24h after their first visit to the store, churn (i.e. never come back to the store again).`

The follow comments were modified:
> endpoint on Vertex AI -> Vertex AI Endpoint
> You will learn -> You learn

> I would consider dropping is aimed at data analysts and data scientists .Seems to unintentional narrow the target audience
Changing to: `This notebook is aimed at users who have data in BigQuery ...`